content/en/blog/_posts/2021-03-18-introducing-suspended-jobs.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Introducing Suspended Jobs"
+date: 2021-03-18
+slug: introducing-suspended-jobs
+url: /blog/2021/03/introducing-suspended-jobs
```

Comment:
I recommend omitting the URL and letting the site generator work out the URL automatically.